patterns/transform-navigation/x-app.html

```diff
@@ -109,8 +109,17 @@
           }
         }
 
-      }
+      },
+
+      observers: ['onLayoutChange(wideLayout)'],
+
+      onLayoutChange: function(wide) {
+        var drawer = this.$.drawer;
 
+        if (wide && drawer.opened) {
```

Comment:
(responding from memory on this)

If the drawer is not open, and layout is !wide the drawer is opened via the menu icon which would negate the need for this check, I believe.